panel/layout/modal.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
```

Comment:
When adding documentation please consider if request https://github.com/awesome-panel/panel-modal/issues/3 should be part of the docs. Thx.